app/Policies/ForumTopicPolicy.php

```diff
@@ -36,23 +39,32 @@ public function view(?User $user, ForumTopic $topic): bool
 
     public function create(User $user, Forum $forum): bool
     {
-        return $user->hasVerifiedEmail();
+        if ($user->isMuted()) {
+            return false;
+        }
+
+        if (!$user->hasVerifiedEmail()) {
+            return false;
+        }
+
+        return true;
     }
 
     public function update(User $user, ForumTopic $topic): bool
     {
-        return $user->is($topic->user);
+        // Muted users might update the topic title (or other properties)
+        // to abuse other users or the platform. We won't allow muted users
+        // to update topics.
+        if ($user->isMuted()) {
+            return false;
+        }
+
+        return $this->manage($user) || $user->is($topic->user);
     }
 
     public function delete(User $user, ForumTopic $topic): bool
     {
-        if ($user->hasAnyRole([
-            Role::MODERATOR,
-        ])) {
-            return true;
-        }
-
-        return $user->is($topic->user);
+        return $this->manage($user) || $user->is($topic->user);
```

Comment:
This seems to be the intent from the left side, but giving the OP permission to delete any topic they've created seems like a dangerous option - particularly when it comes to Official Forum Topics. This might be something to discuss with the moderation team.

Reply:
Agreed, let's retain the current behavior. I've adjusted this so only moderators can delete topics.